docs/components/admin-app/dashboard.md

```diff
@@ -3,8 +3,34 @@
 The __Dashboard__ provided real-time analytics on the performance of your solution. There
 are four time filters available: __Last 24 hours__, __Last week__, __Last month__, and __Last year__.
 
+The dashboard is divided into three sections: __Overview__, __Performance__, and __Insights__.
```

Comment:
How about

```suggestion
The Dashboard provides real-time analytics on the performance of your solution. There
are four time filters available: _Last 24 hours_, _Last week_, _Last month_, and _Last year_.

The dashboard is divided into three sections: [Overview](#overview), [Performance](#performance), and [Insights](#insights).
```

It felt like there were too many emphases!
<img width="776" alt="image" src="https://github.com/user-attachments/assets/7a57dd82-de47-4862-b709-d0708d5b9001">